src/Contexts/StateContext/StateContext.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { createContext, useContext, useEffect, useState } from "react";
+
+const StateContext = createContext();
+
+const ContextProvider = ({ children }) => {
+  const [state, setState] = useState([]);
+  useEffect(() => {
+    (async () => {
```

Comment:
I guess the naming could be more descriptive. Something like ProductContext?

Reply:
Done with suggested changes, Kindly approve PR